site/_docs/reference.md

```diff
@@ -2803,8 +2803,8 @@ In the following:
 | m | JSON_STORAGE_SIZE(jsonValue)                   | Returns the number of bytes used to store the binary representation of *jsonValue*
 | b o p r s | LEAST(expr [, expr ]* )                | Returns the least of the expressions
 | b m p r s | LEFT(string, length)                   | Returns the leftmost *length* characters from the *string*
-| f s | LEN(string)                                  | Equivalent to `CHAR_LENGTH(string)`
-| b f s | LENGTH(string)                             | Equivalent to `CHAR_LENGTH(string)`
+| f r s | LEN(string)                                | Equivalent to `CHAR_LENGTH(string)`
+| b f h p r s | LENGTH(string)                       | Equivalent to `CHAR_LENGTH(string)`
```

Comment:
I think the `r` (from redshift) slipped in this line by mistake?

Reply:
Amazon Redshift supports both `LEN()` and `LENGTH()`.
https://docs.aws.amazon.com/redshift/latest/dg/r_LEN.html

The Redshift function library extends the PostgreSQL function library and by default has all PostgreSQL functions (unless specifically excluded).

Reply:
True! I hadn't realized. Thanks for the clarification.